### PR TITLE
Rename master to main for Beats project

### PR DESCRIPTION
**Pull request by v1v**
### What

Neutral name branch for `elastic/beats`

### Issues

Requires https://github.com/elastic/beats/pull/29710

### CODE REVIEW

**Comment by v1v** [comment]
run elasticsearch-ci/docs

**Comment by v1v** [comment]
run elasticsearch-ci/docs

File: conf.yaml [hunk 1]
```diff
@@ -1927,7 +1927,7 @@ contents:
             current:    *stackcurrent
             index:      x-pack/functionbeat/docs/index.asciidoc
             branches:   [ master, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5 ]
```
Comment (szabosteve) [inline conf.yaml:1929]: Shouldn't be master is `{main: master}` here as well?
```suggestion
            branches:   [ {main: master}, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5 ]
```

File: conf.yaml [hunk 2]
```diff
@@ -2164,9 +2164,9 @@ contents:
           - title:      Beats Developer Guide
             prefix:     en/beats/devguide
             index:      docs/devguide/index.asciidoc
-            current:    master
-            branches:   [ master, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0 ]
-            live:       *stacklive
+            current:    main
```
Comment (v1v) [inline conf.yaml:2167]:
```suggestion
            current:    *stackcurrent
```

**Review by szabosteve — APPROVED** [review]
LGTM!

**Pull request merged**